compiler/rustc_target/src/spec/targets/riscv32e_unknown_none_elf.rs

```diff
@@ -0,0 +1,31 @@
+use crate::spec::{Cc, LinkerFlavor, Lld, PanicStrategy, RelocModel, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    Target {
+        data_layout: "e-m:e-p:32:32-i64:64-n32-S32".into(),
```

Comment:
Sometimes we get PRs that mass-change the `data_layout` string for a bunch of targets, and we had difficulty figuring out why this `data_layout` was being rejected. Can you leave a comment that specifies that RISCV32E's `target.data_layout` here is tied to the `llvm_abiname: "ilp32e"`? Most targets don't use these, and "ilp32e" is close to ilp32f, anyways.

Reply:
Relevant commentary added.